.github/workflows/docker-image.yaml

```diff
@@ -35,5 +35,14 @@ jobs:
       - name: Scan Docker Image
         uses: ./docker-image
         with:
-          service-account-credentials: ${{ secrets.MONDOO_SERVICE_ACCOUNT }}
           image: ${{env.APP}}:${{env.VERSION}}
+        env:
+          MONDOO_CONFIG_BASE64: ${{ secrets.MONDOO_SERVICE_ACCOUNT }}
+      - name: Scan Docker Image
+        uses: ./docker-image
+        with:
+          image: ${{env.APP}}:${{env.VERSION}}
+          args: --score-threshold "10" ${{env.EMPTY_VAR}}
```

Comment:
I don't think we should use `args` as a "catch-all" input for anything other than the "setup" action. Each of the use case actions like Docker, Terraform HCL, k8s-manifest, etc should define very prescriptive inputs like `--score-threshold` 

The `args` input was only intended for the "setup" action as a way for a user to define any supported argument.